### ISSUE TITLE
Feature: Stop training.

### ISSUE THREAD

**Issue by ClothingAI** [cocktailpeanut/fluxgym#206]
A button to stop training. Without having to close the web server.

**Comment by Agiltech-com-au** [comment]
YES PLEASE! omg you get one setting wrong on a cloud machine and such a pain just to STOP and start a new one. so annoying.

**Comment by Agiltech-com-au** [comment]
Haha yeah true. Seems odd to need to reboot server. You can also stop the python script on cloud but a button could be nice

**Comment by muxalko** [comment]
+1

**Comment by jayb2009** [comment]
++1
